examples/extension/README.md

```diff
@@ -34,13 +34,26 @@ For more information, see the `extendedAgent` task in [build.gradle](build.gradl
 
 ## Extensions examples
 
+* Custom `AutoConfigurationCustomizer`: [DemoAutoConfigurationCustomizerProvider](src/main/java/com/example/javaagent/DemoAutoConfigurationCustomizerProvider.java)
 * Custom `IdGenerator`: [DemoIdGenerator](src/main/java/com/example/javaagent/DemoIdGenerator.java)
 * Custom `TextMapPropagator`: [DemoPropagator](src/main/java/com/example/javaagent/DemoPropagator.java)
 * Custom `Sampler`: [DemoSampler](src/main/java/com/example/javaagent/DemoSampler.java)
 * Custom `SpanProcessor`: [DemoSpanProcessor](src/main/java/com/example/javaagent/DemoSpanProcessor.java)
 * Custom `SpanExporter`: [DemoSpanExporter](src/main/java/com/example/javaagent/DemoSpanExporter.java)
 * Additional instrumentation: [DemoServlet3InstrumentationModule](src/main/java/com/example/javaagent/instrumentation/DemoServlet3InstrumentationModule.java)
 
+Configuration to apply select custom classes:
+
+| System property      | Environment variable | Value                                  |
+|----------------------|----------------------|----------------------------------------|
+| otel.traces.exporter | OTEL_TRACES_EXPORTER | com.example.javaagent.DemoSpanExporter |
```

Comment:
did you verify that these values would work or did you just guess that these values should work?

Reply:
the extension project has tests already, can you check those?

Reply:
I've updated the explanation per @mateuszrzeszutek's feedback.  I've also verified that the `otel.propagators` setting is used in the unit tests so I referenced that one specifically where the others are applied dynamically.